### DISCUSSION TITLE
Where can I set the options for @vue/babel-preset-app?

### DISCUSSION THREAD

**Discussion by jessegavin** [vuejs/vue-cli#6913] [Q&A] [answered]
I have a particular need to be able to set different [`targets`](https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/babel-preset-app#targets) for babel-preset-env dynamically from my `vue.config.js` file.

I am unsure how to do that. Is it in the `pluginOptions` like so?

```
module.exports = {
  pluginOptions: {
    '@vue/babel-preset-app': {
      targets: "> 1%, IE 10"
    }
}
```

**Comment by Viggo95** [comment, marked as answer]
You can [specify browsers](https://github.com/browserslist/browserslist#config-file) in `package.json browserslist` or `.browserslistrc` 

A default Vue CLI project uses browserslist field in package.json.


``` json
// package.json
{
  "name": "vue-project",
  "browserslist": [
    "> 1%",
    "last 2 versions",
    "IE 10"
  ],
}
```

@jessegavin

**Reply by jessegavin** [reply]
Thanks, however I am specifically wondering if it's possible to configure this from within my `vue.config.js` file.

**Reply by Viggo95** [reply]
If you specifically want to do this in `vue.config.js`, consider using  [configureWebpack](https://cli.vuejs.org/config/#configurewebpack) and [chainwebpack](https://cli.vuejs.org/config/#chainwebpack)


For example, you can do it like this  👇
- [babel-loader#options](https://github.com/babel/babel-loader#usage)
- [cli-plugin-babel#L108-L125](https://github.com/vuejs/vue-cli/blob/dev/packages/%40vue/cli-plugin-babel/index.js#L108-L125)

**Reply by jessegavin** [reply]
Thank you so much! This is exactly what I was after.

**Reply by Viggo95** [reply]
If  solves your problem, you can mark it as an answer

**Reply by jessegavin** [reply]
I wish I could mark your reply to my reply as the answer.

But I'll mark the initial response as the answer and hopefully someone can find it.

Thanks so much for your help.
